backend/main.py

```
@@ -1,52 +1,49 @@
 from models import User
-from schemas import UserCreate, Token, LoginRequest, UserUpdate
+from schemas import UserCreate, Token, LoginRequest, UserUpdate, OrderOut, OrderCreate
 from utils import auth_utils, create_access_token, verify_token, create_email_verification_token, send_verification_email
 from datetime import timedelta
 import os
 import dotenv
 from pydantic import BaseModel
 import uvicorn
 from typing import Annotated, List
-
+from models import Order, OrderItem, Bill  
```

Ruff (0.8.2)

`models.Order` imported but unused

Remove unused import

(F401)